sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -207,14 +208,11 @@ private static void validateFormWordData(List<TextWord> expectedFormWords,
     }
 
     private static void validateBoundingBoxData(List<Float> expectedBoundingBox, BoundingBox actualBoundingBox) {
```

Comment:
Should this be a `Double` instead of `Float`?

Reply:
This represents the coordinates we get from the service which are `List<Float>`.